components/brave_sync/brave_profile_sync_service_impl.h

```diff
@@ -255,10 +264,14 @@ class BraveProfileSyncServiceImpl
 
   bool pending_self_reset_ = false;
 
+  bool is_model_loaded_observer_set = false;
+
   // Used to ensure that certain operations are performed on the sequence that
   // this object was created on.
   SEQUENCE_CHECKER(sequence_checker_);
 
+  base::WeakPtrFactory<BraveProfileSyncServiceImpl> weak_ptr_factory_{this};
```

Comment:
seems like it is not needed anymore

Reply:
thanks, fixed at https://github.com/brave/brave-core/pull/4588/commits/faae417c8590f9732feec99c5b4df889845ab060